packages/cardano-services/src/cli.ts

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+/* eslint-disable import/imports-first */
+require('../scripts/patchRequire');
+import { API_URL_DEFAULT, OGMIOS_URL_DEFAULT, ProgramOptions, ServiceNames, loadHttpServer } from './program';
+import { Command } from 'commander';
+import { InvalidLoggerLevel } from './errors';
+import { LogLevel } from 'bunyan';
+import { URL } from 'url';
+import { loggerMethodNames } from './util';
+import onDeath from 'death';
+const clear = require('clear');
+const packageJson = require('../package.json');
+
+clear();
+// eslint-disable-next-line no-console
+console.log('Cardano Services CLI');
+
+const program = new Command('cardano-services');
+
+program.version(packageJson.version);
+
+program
+  .command('start-server')
+  .description('Start the HTTP server')
+  .argument('<serviceNames...>', 'List of services to attach')
```

Comment:
A list of options would be nice here

Reply:
Done in 416c09fd72a97dfe9ad74fb71dbb7db94bf0b377